app/src/main/java/com/pluu/support/naver/NaverWeekApi.kt

```diff
@@ -40,8 +38,8 @@ class NaverWeekApi(
                         title = element.select(".toon_name").text()
                         image = element.select("img").first().attr("src")
                         status = when {
-                            element.select("em[class=badge badge_up]").isNotEmpty() -> Status.UPDATE  // 최근 업데이트
-                            element.select("em[class=badge badge_break]").isNotEmpty() -> Status.BREAK  // 휴재
+                            element.select("em[class=badge badge_up]").isNotEmpty() -> Status.UPDATE // 최근 업데이트
+                            element.select("em[class=badge badge_break]").isNotEmpty() -> Status.BREAK // 휴재
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Pluu/WebToon/blob/5475b2548342ec66d67b18c8c8d71379a80ef577/app/src/main/java/com/pluu/support/naver/NaverWeekApi.kt#L42"></span>Exceeded max line length (100) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>